cmd/livepeer/livepeer.go

```diff
@@ -644,11 +644,13 @@ func getOrchWebhook(u string) (*url.URL, error) {
 	if err != nil {
 		return nil, err
 	}
-	if p.Scheme != "http" && p.Scheme != "https" {
-		return nil, errors.New("Webhook URL should be HTTP or HTTP")
+
+	if p.Scheme == "http" || p.Scheme == "https" {
+		glog.Infof("Using orchestrator webhook url %s", u)
+		return p, nil
 	}
-	glog.Infof("Using orchestrator webhook url %s", u)
-	return p, nil
+
+	return nil, errors.New("Webhook URL should be HTTP or HTTP")
```

Comment:
It's generally a bit more idiomatic in golang to exit early on errors, and then proceed with the normal result after errors are handled - so the old flow should be retained.

Also, the original typo is still present here. The error message should say: `"Webhook URL should be HTTP or HTTPS"`

Essentially, this PR should be a one-line commit fixing that error message.

Reply:
> Essentially, this PR should be a one-line commit fixing that error message.

Yes, but when i tried `if p.Scheme != "http" || p.Scheme != "https" {` , `go vet` complained about `suspect or` because this apparently will always return true. 

Also, I didn't wanna commit to your branch, but feel free to do the changes over there and close this one

Reply:
wait a minute, nvm I thought the issue was the condition , not the error msg 🤦‍♂